mipengine/node/monetdb_interface/remote_tables.py

```diff
@@ -12,9 +12,12 @@ def get_remote_tables_names(context_id: str) -> List[str]:
     return common_actions.get_tables_names("remote", context_id)
 
 
-@validate_identifier_names
-def create_remote_table(table_info: TableInfo, url: str):
+# @validate_identifier_names
+def create_remote_table(table_info: TableInfo, url: str, db_name: str):
+
     columns_schema = convert_schema_to_sql_query_format(table_info.schema)
-    cursor.execute(
-        f"CREATE REMOTE TABLE {table_info.name} ( {columns_schema}) on '{url}' WITH USER 'monetdb' PASSWORD 'monetdb'")
+    query = f"CREATE REMOTE TABLE {table_info.name} ( {columns_schema}) ON 'mapi:monetdb://{url}/{db_name}' WITH USER 'monetdb' PASSWORD 'monetdb'"
```

Comment:
I think you are wrong about the definition of url. [Wikipedia](https://en.wikipedia.org/wiki/URL) says otherwise.